hmy/hmy.go

```diff
@@ -116,6 +116,9 @@ type NodeAPI interface {
 	GetConfig() commonRPC.Config
 	ShutDown()
 	GetLastSigningPower() (float64, error)
+
+	// mmr APIs
+	// GetProof(txHash common.Hash, blockHash common.Hash, blockNumber, withRespectTo uint64) mmr.MmrProof
```

Comment:
why is this commented?

Reply:
will remove it. it's the old MMR-proof RPC function.